browser/ui/webui/brave_federated/federated_internals_page_handler.h

```diff
@@ -6,19 +6,19 @@
 #ifndef BRAVE_BROWSER_UI_WEBUI_BRAVE_FEDERATED_FEDERATED_INTERNALS_PAGE_HANDLER_H_
 #define BRAVE_BROWSER_UI_WEBUI_BRAVE_FEDERATED_FEDERATED_INTERNALS_PAGE_HANDLER_H_
 
-#include <string>
-#include <vector>
-
 #include "base/memory/raw_ptr.h"
 #include "base/memory/weak_ptr.h"
 #include "brave/browser/ui/webui/brave_federated/federated_internals.mojom.h"
-#include "brave/components/brave_federated/data_store_service.h"
-#include "brave/components/brave_federated/data_stores/ad_notification_timing_data_store.h"
+#include "brave/components/brave_federated/data_stores/data_store.h"
```

Comment:
is this used?

Reply:
used for `brave_federated::TrainingData`